src/core/components/AMInstallButton/index.js

```diff
@@ -45,23 +45,23 @@ import type { ReactRouterLocationType } from 'core/types/router';
 import './styles.scss';
 
 type Props = {|
+  ...WithInstallHelpersInjectedProps,
   addon: AddonType,
   className?: string,
   defaultInstallSource: string,
   disabled: boolean,
-  location: ReactRouterLocationType,
```

Comment:
should you setCurrentStatus that you have added above?

Reply:
it is part of `WithInstallHelperInjectedProps`.

Reply:
ahh..I see..cool 👍